### DISCUSSION TITLE
data/nativetest/TFLiteRunner/TFLiteRunner

### DISCUSSION THREAD

**Discussion by Juan52122** [microsoft/vscode-discussions#2403] [Extension Development QnA] [answered]
/data/nativetest/TFLiteRunner/TFLiteRunner
/data/nativetest64/TFLiteRunner/TFLiteRunner
This MediaTek software package contains software with the following notices and under the following licenses:

==============================================================================================================

Copyright (C) 2011  Carl Rogers

The MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

**Comment by isidorn** [comment, marked as answer]
I do not understand this question. Thus I am going to go ahead and close it. If you have some clarification please comment. Thanks
